### ISSUE TITLE
Pencil cannot read this file

### ISSUE THREAD

**Issue by infinityglitches** [pencil2d/pencil#862]
I'm done, I give up. I cant use this system anymore.

I tried loading up my animation, it said "Pencil cannot read this file, If you want to import images use the command import.

Are you kidding me , I spent DAYS on this animation and now it wont let me access my animation.
I understand this is still beta and has a lot of bugs but this is too much.

Bye, Pencil 2D community.

**Comment by Jose-Moreno** [comment]
@infinityglitches Sorry to hear you had this issue. But would you mind trying and explain what were you doing before getting this problem? Like did you get a crash? or did you save the file and there was a blackout?
 Or were you trying to open the file in a different program?. What version of Pencil2d you are using?. etc

All of this info is essential for the developers to try and fix whatever problems there are. Besides no computer software exists without bugs to fix, Pencil2D is no different than say Adobe Flash in that regard. 

If you feel inclined to help the developers understand what happened to help prevent this issue for others, let us know. On the other hand, since you were trying to embed mp3 files you converted somehow, this is what might have cause the corruption issue. If you want try to upload the file and paste the link here so the developers can take a look and see if they can fix the file for you.

Lastly if you really want a kind of stable program, as a professional advice I suggest you think of purchasing an animation package if you intent to continue creating animation either for an extended hobby or for a living. At least you'll have the benefit of having dedicated technical support for the money you pay.

If you want a simple program like Pencil2D go and try out Digicel Flipbook https://digicel.net/download/ Mind you it's mostly for hand drawn animation, so there's no tweening.

If you want something a bit more powerful, and with vector try moho 12 debut version which is rather cheap, and more novice friendly that the pro version http://my.smithmicro.com/anime-studio-debut.html

Take care and hope you still show us your work made in the near future. Peace.

**Comment by infinityglitches** [comment]
Hi, I'm using this for a YouTube channel as a hobby, I'm just looking for something casual to use.
I exited out of Pencil 2D to go get lunch, I came back, loaded up Pencil 2D, loaded up my animation and got the message. lol I'm just a 15 year old making vids for fun.

**Comment by infinityglitches** [comment]
also I'm using v0.6.0

**Comment by Jose-Moreno** [comment]
@infinityglitches Yeah no problem, I know many youtubers that are doing it for fun, and they still have purchased their tools. There's no harm in that. If you just want free tools. Maybe a combination of Krita and a video editor might help you out. Krita is basically a free photoshop, it has an animation timeline too and you can export image sequences and videos (with some effort to configure it the first time)

The video editor is for getting all the sounds and exporting the final video with the sound effects and stuff.

Regarding the Pencil2D thing, it could be because of the sound file, or because of using the "open recent" menu. If you try to open through "File > Open" does it get you the same error? if it does, then again I urge you to upload the file somewhere and link it. If you want you can send it privately to pencil2danimation (at) gmail.com (at) -> @ and it will only be accessible by the team members.

**Comment by infinityglitches** [comment]
Thank you for the help, I will try out Krita, and if that doesn't work ill link my animation. Thanks again!

**Comment by Jose-Moreno** [comment]
@infinityglitches No problem. Well you can still link the file to see if we can salvage your work, so you don't have to remake it from scratch. Ah well there's also something you can do.

You see .pclx files are really like zip files. So if you create a copy and rename it with the .zip extension you can open it like a normal compressed file. If there's a file with folders and other files inside, there's a high chance your animation is still working. You'd have to extract the folders you would be able to find the your images inside, although they would be disorganized if you imported just the images on a different software, im sure you could reposition them easily.

**Comment by Jose-Moreno** [comment]
Anyway, good luck with all your stuff. Let us know if you still want us to try and salvage the file, even if there's no guarantee it's better than having to redo everything.

**Comment by infinityglitches** [comment]
OK Krita is too complicated, how do I link a file? or how do I copy and extract it.

**Comment by Jose-Moreno** [comment]
@infinityglitches You can upload it to dropbox, mega.nz, google drive, or similar services. Then the public link can be pasted here. Or if you want more privacy send it to the email I mentioned before and let us know when you send it so someone can check the email later on.

**Comment by infinityglitches** [comment]
ok thx

**Comment by CandyFace** [comment]
You can also zip the file https://support.microsoft.com/en-us/help/14200/windows-compress-uncompress-zip-files

then drag it into the comment area, or click the "selecting them" and choose the compressed zip file.

It would be very helpful if you would upload your project, so we have a chance preventing this from happening in the future. By doing this there's also a chance that we can recover your work and at the same time you're also helping us making the software better.

**Comment by infinityglitches** [comment]
Thanks everyone for the help, i figured out that it was the mp3 file that didn't allow me to access my file, so I will make a new animation with no music, like a moving comic. Thanks everyone!  and sorry for the trouble.

Also, my computer doesn't let me put my file in this comment section, it says "We don't support that file type", I even tried using Zamzar to convert and it said the same thing.

Thanks Again!

**Comment by CandyFace** [comment]
You don't have to make an animation without sound, just import a proper sound file and everything should be fine. Did you compress the file project to a zip? you can't upload a .pclx project here, that's why you have compress it like in the guide I showed above. 

As for the sound file, here's a guide to how you convert it to .wav
 
* open audacity
* goto: file -> "open"
* select your (assuming) .wma file
* a popup may show, just press OK

Now you should have the file opened in audacity, all you need now is to export it as a .wav.
Do the following:
* goto: file -> "export audio"
* select .wav as format.
* save.

Now you can import the sound into Pencil2D.

**Comment by chchwy** [comment]
Hey @infinityglitches please upload your project file, either by dropbox or google drive.
If we can figure out what happened and fix it, it will prevent another user from running into the same problem. And thanks for your reporting, any bug report is making pencil2d better and more solid.

**Comment by Kaikogf** [comment]
Hey developers.

@infinityglitches is right about his complains. I happened to be in the same situation a minute ago and I thought that my animation got lost. The same error message showed up. 

I tried different animations that I had saved before (.pclx). I noticed the following. Hope this help somehow **_(build 17Jan2018)_**

1- I had the same exact animation but with different names, Pencil2D didn't open [File > Open] the one with a special symbol so I change _**cópia**_ for _**copia**_ and did work.
2- I tested Pencil2D again with some other animations and I realized that the problem is in the build 17Jan2018 since it did not allow me to open all my animations but only some of them (not sure what is inside the animation to be opened)
3- Changing the build and testing the animations with version 31/12/2017 everything worked OK

**Comment by Jose-Moreno** [comment]
@Kaikogf Thanks for reporting. However, and I don't want this to sound annoying, they definitely can't do anything if people don't share their files. It is very difficult to reproduce these kind of bugs because there's a lot of variables involved in how a user does something. So please always try to share the file.

In your specific case, according to what you're saying it seems there's a problem with character encoding. So it means that Pencil2d can't read files that have weird characters, which in Spanish, Portuguese, French and other languages are an absolute part of them like the _tilde_.

Also thanks for letting us know which builds you tested have a great weekend! :smile:

**Comment by Kaikogf** [comment]
So here it is the Pencil2D's file titled "MyAnimation" with just one layer and scribbles along 12 frames, which works OK with **_version 31/12/2017_**, but failed to load with **_build 17Jan2018_** (error _"Pencil cannot read this file, If you want to import images use the command import"_)

[MyAnimation.zip](https://github.com/pencil2d/pencil/files/1649514/MyAnimation.zip)

In my previous comment I talked about the issue of using special letters (ex: á). Just want to clarify that all other animations I tested and failed to load where not because of "weird" characters !, like this one MyAnimation.
By the way, this kind of special characters should be allowed in Pencil2D. We have came so far as to translate Pencil in several languages so we can easily understand that non-English-speaker will use their own special characters to give their animations a name and so does not have to guess if it works OK or not because of a letter.

Thank you @Jose-Moreno ... you are helping Developers and Community a great deal!👍
